src/components/Modal.js

```diff
@@ -27,11 +28,30 @@ const styles = {
 
 const Modal = ({ onDismiss, title, titleExtras, children, width = 450, showCancel = true, cancelText = 'Cancel', showX, showButtons = true, okButton, ...props }) => {
   const titleId = Utils.useUniqueId()
+  const modalNode = useRef()
+  const previouslyFocusedNode = useRef()
+
+  Utils.useOnMount(() => {
+    previouslyFocusedNode.current = document.activeElement
+    return () => previouslyFocusedNode && previouslyFocusedNode.current && previouslyFocusedNode.current.focus()
```

Comment:
nit: The ref object itself should always be present, so I think this can be:
```suggestion
    return () => previouslyFocusedNode.current && previouslyFocusedNode.current.focus()
```